src/utils/global/ambassadorCard/ambassadorCard.module.scss

```diff
@@ -1,16 +1,18 @@
 @import '../../../variables.scss';
 
 .ambassadorCard {
-  width: 295px;
-  height: 460px;
-  max-height: 100vh;
+  width: 18.5rem;
+  max-width: 100%;
+  min-height: min(28.5rem, 100%);
+  max-height: 100%;
+
   justify-content: start;
   align-items: baseline;
```

Comment:
I think this should be `flex-start`. Right now the content get offset to the right a bit.

Reply:
I went with `align-items: start` -- I was developing in Firefox where it seems this wasn't an issue, but yeah, in Chrome it was misaligned with the scrollbar -- very odd.